analysis/topeft_run2/analysis_processor.py

```diff
@@ -479,14 +479,20 @@ def process(self, events):
             j0 = goodJets[ak.argmax(goodJets.pt,axis=-1,keepdims=True)]
 
             # Loose DeepJet WP
-            loose_tag = "btag_wp_loose_" + year.replace("201", "UL1")
+            if btagAlgo == "btagDeepFlavB": 
+                loose_tag = "btag_wp_loose_" + year.replace("201", "UL1")
+            if btagAlgo == "btagPNetB":
+                loose_tag = "btag_wp_loose_PNet_" + year.replace("201", "UL1")
```

Comment:
I did add this module a while back if it helps simplify this logic block
https://github.com/TopEFT/topcoffea/blob/main/topcoffea/modules/run_id.py